### DISCUSSION TITLE
Can we use compression on Parquet writing?

### DISCUSSION THREAD

**Discussion by metesynnada** [apache/datafusion#6061] [Q&A] [answered]
Is there a `parquet.gz` etc file type that we would use compression?

**Comment by tustvold** [comment, marked as answer]
Parquet natively supports block level compression - see https://github.com/apache/parquet-format/blob/master/Compression.md

Compressing the file as a whole would lose the ability to do meaningful IO pushdown, and would likely yield minimal size savings, so I'm not aware of anyone doing this
